lab1/task 1/Matrix3x3.cpp

```diff
@@ -0,0 +1,77 @@
+#include "Matrix3x3.h"
+
+Matrix3x3::Matrix3x3() 
+{
+    for (int i = 0; i < 3; i++)
+        for (int j = 0; j < 3; j++)
+            matrix[i][j] = 0;
+}
+
+void  Matrix3x3::fillRandomElements(const int minVal, const int maxVal) 
+{
+    for (int i = 0; i < 3; i++)
+    {
+        for (int j = 0; j < 3; j++)
+        {
+            matrix[i][j] = rand() % (maxVal-minVal) + minVal;
+        }
```

Comment:
```suggestion
    for (int i = 0; i < 3; i++)
        for (int j = 0; j < 3; j++)
            matrix[i][j] = rand() % (maxVal-minVal) + minVal;
```
Так более лаконично.